src/app/publications/page.tsx

```diff
@@ -105,74 +134,78 @@ export default function Page() {
-          />
-          <Filter
-            filterName="Publication Type"
-            optionSet={['All', ...PublicationTypes]}
-            optionSelected={publicationType}
-            handleOptionChange={handlePublicationTypeChange}
+    <CenteredContainer>
+      <Container>
+        <main style={{ padding: '0px', margin: '0px' }}>
+          <h1>Publications</h1>
+          <Filters>
+            <Filter
+              filterName="Research Topic"
+              optionSet={['All', ...Object.keys(ResearchTopics)]}
+              optionSelected={researchTopic}
+              handleOptionChange={handleResearchTopicChange}
+            />
+            <Filter
+              filterName="Publication Type"
+              optionSet={['All', ...PublicationTypes]}
+              optionSelected={publicationType}
+              handleOptionChange={handlePublicationTypeChange}
+            />
+          </Filters>
+          <Sections>
+            {publicationList.filter(pub => pub.type === 'preprint').length > 0 && (
+              <Section
+                id="preprints"
+                ref={el => {
+                  sectionRefs.current['preprints'] = el
+                }}
+              >
+                <SectionTitle>Preprints</SectionTitle>
+                <SectionContent>
+                  {publicationList
+                    .filter(pub => pub.type === 'preprint')
+                    .map(pub => (
+                      <PublicationCard key={pub.title} pub={pub} />
+                    ))}
+                </SectionContent>
+              </Section>
+            )}
+            {uniq(publicationList.map(p => p.year))
+              .sort()
+              .reverse()
+              .map((year, i) => (
+                <React.Fragment key={year}>
+                  {i === 0 && publicationList.filter(pub => pub.type === 'preprint').length === 0 ? null : (
+                    <Divider key={`divider-${i}`} />
+                  )}
+                  <Section
+                    id={`year-${year}`}
+                    ref={el => {
+                      sectionRefs.current[`year-${year}`] = el
+                    }}
+                    key={year}
+                  >
+                    <SectionTitle>{year}</SectionTitle>
+                    <SectionContent>
+                      {publicationList
+                        .filter(({ year: y }) => y === year)
+                        .map(pub => (
+                          <PublicationCard key={pub.title} pub={pub} />
+                        ))}
+                    </SectionContent>
+                  </Section>
+                </React.Fragment>
+              ))}
+          </Sections>
+        </main>
+        <SideContainer>
+          <Sidebar
+            activeSection={activeSection}
+            handleLinkClick={handleLinkClick}
+            publicationList={publicationList.map(pub => ({ year: pub.year, type: pub.type }))}
```

Comment:
```suggestion
            publicationList={publicationList}
```
since js use duck typing, creating another list with `map` is unnecessary.